lib/IncomingStreamTrackBridge.js

```diff
@@ -51,6 +51,7 @@ const IncomingStreamBridge = require("./IncomingStreamBridge");
  * 
  * @property {number} [rtt] Round Trip Time in ms
  * @property {number} bitrate Bitrate for media stream only in bps
+ * @property {number} totalBitrate average bitrate (media + RTP overhead) received during last second in bps
  * @property {number} total Accumulated bitrate for rtx, media streams in bps
```

Comment:
make `total` deprecated
 
```suggestion
 * @deprecated @property {number} total Accumulated bitrate in bps
```